test/games/strategy/triplea/delegate/LHTRTest.java

```diff
@@ -227,6 +227,48 @@ public Object invoke(final Object proxy, final Method method, final Object[] arg
     // Changed to match StrategicBombingRaidBattle changes
     assertEquals(PUsBeforeRaid - 8, PUsAfterRaid);
   }
+
+  @Test
+  public void testBombingRaid2targets() {
+    final Territory germany = m_data.getMap().getTerritory("Germany");
```

Comment:
IE: please never copy paste, instead move existing code to a method and parameterize it first. There are other ways to get the same effect/abstraction, but a parameterized method is generally most straight forward.